.gitignore

```diff
@@ -336,3 +336,5 @@ ASALocalRun/
 DebugPlatforms.props
 .DS_Store
 **/.icons
+
+!build/ci/publish
```

Comment:
Every time I see this `!` in a `gitignore` file it takes me some time to remember what does it mean. 😅